runtime/src/registry.rs

```diff
@@ -424,6 +419,12 @@ pub fn get_user_with_account(account_id: AccountId) -> Option<User> {
         .map(|(id, user)| User::new(id, user))
 }
 
+pub fn find_org(predicate: &dyn Fn(&state::Org) -> bool) -> Option<state::Org> {
+    store::Orgs::iter()
+        .find(|(_, org)| predicate(org))
+        .map(|(_, org)| org)
```

Comment:
Can't this be merged into a [`FilterMap`](https://doc.rust-lang.org/std/iter/struct.FilterMap.html).

Are we planning to optimise this at some point, it seems expensive to iterate over the entire org space every time we do this check.

Reply:
TIL, should do that

Reply:
actually, opting for `filter_map` here results in more cumbersome code:

```suggestion
    .find_map(|(_, org)| if predicate(&org) { Some(org) } else { None })
```

Given that in this case the `map` only once, I'd rather keep this cleaner, more declarative structure.

Reply:
> Are we planning to optimise this at some point, it seems expensive to iterate over the entire org space every time we do this check.

Indeed, this is an expensive operation of O(n*m) complexity, where `n` is the number of orgs and `m` the number of members per org. However, considering the cost of storing stuff on-chain, I'm not convinced we need to look after this right now nor in the near future.